src/command_modules/azure-cli-network/azure/cli/command_modules/network/mgmt_express_route_circuit/azuredeploy.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "bandwidthInMbps": {
+      "type": "int",
+      "metadata": {
+        "description": "Bandwidth in Mbps of the circuit being created. It must exactly match one of the available bandwidth offers List ExpressRoute Service Providers API call."
+      }
+    },
+    "circuitName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the ExpressRoute circuit"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for resources."
+      }
+    },
+    "peeringLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the peering location. It must exactly match one of the available peering locations from List ExpressRoute Service Providers API call."
+      }
+    },
+    "serviceProviderName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the ExpressRoute Service Provider. It must exactly match one of the Service Providers from List ExpressRoute Service Providers API call."
+      }
+    },
+    "sku_family": {
+      "type": "string",
+      "defaultValue": "MeteredData",
+      "allowedValues": [
+        "MeteredData",
+        "UnlimitedData"
+      ],
```

Comment:
Right, so these enums are set to "ModelAsString=True" in the Swagger. Thus: 1) they should not have allowedValues in the ARM template. 2) They should use completers instead of choices in the parameter aliasing. 3) The help text should list some or all of the "allowedValues".  I've already got an issue opened against the Swagger... feel free to vote it up!!